file-formats/dcat/dcat-v1.json

```
@@ -77,7 +77,8 @@
           "enumDescriptions": [
             "Sensitive personal data",
             "Personal data"
```

This is only cosmetic: Did you think of having the default value as first value? 
```suggestion
             "Personal data",
            "Sensitive personal data"
```
This would also mean switching the component in the constant but I do not expect any problems with this change.

---

Change the sequence as suggested.